.env

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_SIMPLEJSONCMS_ID=mdb18slfe7
```

Comment:
i guess without the password nobody can do anything with this?

Reply:
yeah the password is what lets people modify the CMS, this is just used for GET requests